index.js

```diff
@@ -171,6 +172,7 @@ function fastifyView (fastify, opts, next) {
       return
     }
 
+    data = Object.assign(defaultCtx, data)
```

Comment:
Wouldn't it be better to `Object.assign({}, defaultCtx, data)`? This way you do not mutate the object passed in.

Reply:
fixed